edx_sga/sga.py

```diff
@@ -449,27 +469,43 @@ def prepare_download_submissions(self, request, suffix=''):  # pylint: disable=u
         user = self.get_real_user()
         require(user)
         zip_file_ready = False
+        location = unicode(self.location)
 
         if self.is_zip_file_available(user):
+            log.info("Zip file already available for block: %s for staff: %s", location, user.username)
             assignments = self.get_sorted_submissions()
             if assignments:
                 last_assignment_date = assignments[0]['timestamp'].astimezone(pytz.utc)
+                log.info(
+                    "Last assignment upload time: %s for block: %s for staff: %s",
+                    last_assignment_date,
+                    location,
+                    user.username
+                )
                 zip_file_path = get_zip_file_path(
                     user.username,
                     self.block_course_id,
                     self.block_id,
                     self.location
                 )
                 zip_file_time = get_file_modified_time_utc(zip_file_path)
+                log.info(
+                    "Last assignment upload time: %s, last zip file time: %s for block: %s for staff: %s",
```

Comment:
1. ~~`for staff:`~~ --> `for user:` or `for instructor:`
1. ~~`last zip file time:`~~ --> `zip file modified time:`